noah/src/main/scala/edu/uci/ics/cloudberry/noah/parser/HTMLparser.java

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.cloudberry.noah.parser;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import java.io.*;
+
+public class HTMLparser {
+
+    public static String parse(String dirc) throws IOException {
+        File f = new File(dirc);
+        String info = null;
+        if (f.isFile()) {
+            try {
+                Document doc = Jsoup.parse(f, "UTF-8");
+                info = doc.getElementById("preview").text();
+                if (info.intern() == "") {
+                    throw new Exception("empty file");
+                }
+                return info;
+            } catch (Exception e) {
+                System.out.println("catch an exception");
```

Comment:
should print out the exception by `e.printStacktrace` to show the exact problem.